x-pack/plugins/cases/public/components/create/custom_fields.tsx

```diff
@@ -41,6 +45,17 @@ const CustomFieldsComponent: React.FC<Props> = ({ isLoading, customFieldsConfigu
 
   return (
     <EuiFlexGroup direction="column" gutterSize="s">
+      {customFieldsComponents.length ? (
+        <EuiText
+          size="m"
+          css={css`
+            font-weight: ${euiTheme.font.weight.bold};
+          `}
+        >
+          {i18n.ADDITIONAL_FIELDS}
```

Comment:
Missed this header from [figma](https://www.figma.com/file/bDw4jOaOy8ke3hL6uYrfRK/Custom-Fields?type=design&node-id=601-341410&mode=design&t=J9dIYEtYN1tnMhZj-0)

Reply:
What about using an `h3` or `h4` and playing with the `size` instead of using CSS?